README.md

```diff
@@ -1,4 +1,14 @@
-# The OpenJS Foundation Cross Project Council
+<p align="center">
+  <br />
+  <a href="https://openjsf.org">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color-dark_background.svg">
+      <img src="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color.svg" width="200px">
+    </picture>
+  </a>
+</p>
+
+<h1>Cross Project Council</h1>
```

Comment:
```suggestion
# Cross Project Council
```
this can still use markdown

Reply:
Unless we center it. What do you think?

Reply:
Also not keen on aligning the text, but it feels weird to have the logo floating center and the text on the left. IMO, the text should be on the left.

I know you just ctrlc/ctrlv what I've done on the nodejs.org repo, but there's no need to center the logo here, IMHO.